src/Lti/Service/GetUserAssignmentLtiRequestService.php

```diff
@@ -140,7 +140,13 @@ private function getAssignmentLtiLink(Assignment $assignment): string
         return sprintf(
             '%s/%s',
             $link,
-            base64_encode((string)json_encode(['delivery' => $assignment->getLineItem()->getUri()]))
+            base64_encode(
+                json_encode(
+                    ['delivery' => $assignment->getLineItem()->getUri()],
+                    JSON_THROW_ON_ERROR,
```

Comment:
As you changed the json_encode output error. Please make sure that they are well caught.

Reply:
It is caught by the framework, or what do you mean here?